conda/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set data= load_setup_py_data() %}
 package:
-  name: ts-athexapod
+  name: "ts-athexapod"
```

Comment:
Please revert this change. yaml files need not double quotes around string constants unless they contain a few unusual characters.